modules/openthread/platform/radio.c

```diff
@@ -848,6 +849,33 @@ otError platformRadioTransmitCarrier(otInstance *aInstance, bool aEnable)
 }
 #endif /* CONFIG_IEEE802154_CARRIER_FUNCTIONS */
 
+otError platformRadioTransmitModulatedCarrier(otInstance *aInstance, bool aEnable,
```

Comment:
should this be guarded with some configs? like `CONFIG_IEEE802154_CARRIER_FUNCTIONS` ?

Reply:
fixed. thanks